recipes/libaom-av1/all/conanfile.py

```diff
@@ -91,6 +91,10 @@ def package(self):
         cmake = CMake(self)
         cmake.install()
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        if self.settings.os == "Windows" and os.path.exists(os.path.join(self.package_folder, "lib", "aom_dll.lib")):
+            # Make the names of aom_dll.lib and aom.dll consistent with each other and other platforms
+            os.rename(os.path.join(self.package_folder, "lib", "aom_dll.lib"),
```

Comment:
These renamos always come back to bite us in the most unexpected ways, so I'm always wary of them

Reply:
@valgur please, do not rename it. Follow what's expected from the upstream.

Reply:
Uh, yes, that was not a great idea. I'll revert it.